src/containers/pages/InterventionPlan/PlanDefinitionList/index.tsx

```diff
@@ -33,6 +33,9 @@ const PlanDefinitionList = (props: PlanListProps) => {
   const { fetchPlans, plans, service } = props;
   const [loading, setLoading] = useState<boolean>(true);
 
+  /** Track status of the component  during a single render */
+  let isMounted = true;
+
   const apiService = new service('/plans');
```

Comment:
@moshthepitt  should this be `plans` or `/plans`?

Reply:
it should be `plans`.  Does a test exist that checks either we call the service correctly, or that the URL mock call is as expected? @p-netm

Reply:
yeap, will make sure there are there